### DISCUSSION TITLE
Add Pagination Metadata `<link>` tag `rel="prev | next"` to the `generateMetadata` options.

### DISCUSSION THREAD

**Discussion by devjiwonchoi** [vercel/next.js#52926] [Ideas]
### Goals

Add a pagination option to the `generateMetadata`  [Metadata Field](https://nextjs.org/docs/app/api-reference/functions/generate-metadata#metadata-fields) as follows:

```js
pagination: {
  prev: string,
  next: string,
}
```

Which will generate `<link>` tag as:

```html
<link rel="prev" href="https://www.example.com/article?story=abc&page=1" />
<link rel="next" href="https://www.example.com/article?story=abc&page=3" />
```

### Background

W3C docs support for `<link>` tag `rel="prev | next"`.

Since it is a minor feature, this option could easily be replaced by adding a custom `<head>` tag.

```js
<html>
  <head>
    <link rel="prev" key="prev" href="someURL" /> // here
    <link rel="prev" key="next" href="someURL" /> // here
  </head>
  <body>{children}</body>
</html>
```

### Reference

- [HTML `<link>` rel Attribute](https://www.w3schools.com/tags/att_link_rel.asp)

### Proposal

Extends #49919

I am willing to add this feature as a contributor.

### PR

#53184

**Comment by vadym-lohvynovych** [comment]
It would be really nice! I could also help with contribution. Website I'm developing is SEO oriented ind it's important for up to have these links.

**Reply by devjiwonchoi** [reply]
@vadym-lohvynovych I've opened a PR #53184 and GitHub linked it to #49919 😅. I'll add it to this issue, my bad.
